docs/docsite/rst/dev_guide/ansible_index.rst

```diff
@@ -75,6 +75,7 @@ If you prefer to read the entire guide, here's a list of the pages in order.
    developing_python_3
    debugging
    developing_modules_documenting
+   adjacent_yaml_doc
```

Comment:
@oraNod  @bcoca FWIW this won't show up on Ansible package docs as it's an incorrect file name (it's sidecar). I'm guessing CI doesn't detect problems like this but we need another PR to correct this pls.

Reply:
pr at https://github.com/ansible/ansible/pull/79772